### ISSUE TITLE
NextJS Version 13.5 brokes

### ISSUE THREAD

**Issue by jspasiuk** [froala/react-froala-wysiwyg#381]
We just updated NextJS to the latest version (13.5) from 13.4 and froala started crashed (some type issue)

`import "froala-editor/css/froala_style.min.css";
import "froala-editor/css/froala_editor.pkgd.min.css";
import FroalaEditorComponent from "react-froala-wysiwyg";
import "froala-editor/js/plugins/font_size.min.js";
import "froala-editor/js/plugins/align.min.js";
import config from "../../../config";

export default function CustomTextEditor({
  initialValue,
  setProp,
}: {
  initialValue: string;
  setProp: any;
}) {
  return (
    <FroalaEditorComponent
      config={{
        key: config.FROALA_KEY,
        initOnClick: true,
        disableRightClick: true,
        toolbarInline: true,
        charCounterCount: false,
        toolbarVisibleWithoutSelection: true,
        reactIgnoreAttrs: ["class", "id"],
        htmlAllowedTags: ["p"],
      }}
      onModelChange={(model: any) => {
        setProp((props: any) => (props.initialValue = model));
      }}
      tag="div"
      model={initialValue}
    />
  );
}
`

This is our code, and we import the editor using 

`
const CustomTextEditor = dynamic(() => import("./CustomTextEditor.component"), {
  ssr: false,
});`

Thanks

**Comment by TylerNRobertson** [comment]
+1 on this, same for me in our project as well

**Comment by TylerNRobertson** [comment]
Seems to only happen in a production optimized build, in development mode it works fine

**Comment by Luyin** [comment]
It's works on NextJS v13.4... hmm

**Comment by TylerNRobertson** [comment]
@Luyin Yeah this issue is for 13.5, its weird cus it works in 13.5.3 but not any minor version after that

**Comment by ilyaskarim** [comment]
> We just updated NextJS to the latest version (13.5) from 13.4 and froala started crashed (some type issue)
> 
> ```
> import "froala-editor/css/froala_style.min.css";
> import "froala-editor/css/froala_editor.pkgd.min.css";
> import FroalaEditorComponent from "react-froala-wysiwyg";
> import "froala-editor/js/plugins/font_size.min.js";
> import "froala-editor/js/plugins/align.min.js";
> import config from "../../../config";
> 
> export default function CustomTextEditor({
>   initialValue,
>   setProp,
> }: {
>   initialValue: string;
>   setProp: any;
> }) {
>   return (
>     <FroalaEditorComponent
>       config={{
>         key: config.FROALA_KEY,
>         initOnClick: true,
>         disableRightClick: true,
>         toolbarInline: true,
>         charCounterCount: false,
>         toolbarVisibleWithoutSelection: true,
>         reactIgnoreAttrs: ["class", "id"],
>         htmlAllowedTags: ["p"],
>       }}
>       onModelChange={(model: any) => {
>         setProp((props: any) => (props.initialValue = model));
>       }}
>       tag="div"
>       model={initialValue}
>     />
>   );
> }
> ```
> 
> This is our code, and we import the editor using
> 
> ` const CustomTextEditor = dynamic(() => import("./CustomTextEditor.component"), { ssr: false, });`
> 
> Thanks


Can we get more details about the error you're encountering? Additionally, we've made enhancements to our Type definitions, so you might want to try our latest version to see if it resolves the issue.
